layouts/partials/project-navbar.html

```diff
@@ -5,19 +5,21 @@
 <div class="container">
     <div class="row row-centered">            
         <div class="col-xs-6 col-centered col-max">
-            <a href="/{{ .Params.projectSlug }}/">Get Started</a>
+            <a href="/{{ .Params.projectslug }}/">Get Started</a>
         </div>
-        <!--<div class="col-xs-6 col-centered col-max">
-            <a href="/{{ .Params.projectSlug }}/developer-guide/">Developer Guide</a>
-        </div>-->
+        {{ if ne .Params.projectslug "android" }}
```

Comment:
That branch looks like the right direction to take the layouts. Let's take this up as a separate pull request, which certainly may be what you had in mind.